build error in "/usr/include/pngconf.h:372" on Debian ?

Hello, it seems there is a regression in the build process, at least on Debian **stretch** . I get this error which I don't understand and which I did not have before: 

```bash
$ python setup.py build
[...]
x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fdebug-prefix-map=/build/python3.5-3.5.3=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -D_REENTRANT -DENABLE_NEWBUF=1 -I/usr/X11R6/include -I/usr/include/SDL -I/usr/include/SDL -I/usr/include -I/usr/include -I/usr/include/python3.5m -I/home/eea/src/sand_3.5/include/python3.5m -c src_c/imageext.c -o build/temp.linux-x86_64-3.5/src_c/imageext.o
In file included from /usr/include/png.h:540:0,
                 from src_c/imageext.c:36:
/usr/include/pngconf.h:371:21: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘.’ token
            __pngconf.h__ in libpng already includes setjmp.h;
                     ^
/usr/include/pngconf.h:372:12: error: unknown type name ‘__dont__’
            __dont__ include it again.;
            ^~~~~~~~
/usr/include/pngconf.h:372:29: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘it’
            __dont__ include it again.;
                             ^~
/usr/include/pngconf.h:372:29: error: unknown type name ‘it’

$ uname -a
Linux terbium 4.9.0-8-amd64 #1 SMP Debian 4.9.130-2 (2018-10-27) x86_64 GNU/Linux

$ python --version
Python 3.5.3
```
 I couldn't find any png-lib on [CompileUbuntu](https://www.pygame.org/wiki/CompileUbuntu?parent=) , so I wunder how the problem comes up.

--- a/src_c/imageext.c
+++ b/src_c/imageext.c
@@ -30,17 +30,6 @@
 
 #if IS_SDLv1
 
-#ifdef __SYMBIAN32__ /* until PNG support is done for Symbian */
-#include <stdio.h>
-#else
-#include <png.h>
-#endif
-
-#include <jerror.h>
-#include <jpeglib.h>
-
-#endif /* IS_SDLv1 */
-
 /* Keep a stray macro from conflicting with python.h */
 #if defined(HAVE_PROTOTYPES)
 #undef HAVE_PROTOTYPES
@@ -52,6 +41,19 @@
 #if defined(HAVE_STDLIB_H) /* also defined in pygame.h (SDL.h) */
 #undef HAVE_STDLIB_H
 #endif
+
+#ifdef __SYMBIAN32__ /* until PNG support is done for Symbian */
+#include <stdio.h>
+#else
+// PNG_SKIP_SETJMP_CHECK : non-regression on #662 (build error on old libpng)
+#define PNG_SKIP_SETJMP_CHECK
+#include <png.h>
+#endif
+
+#include <jerror.h>
+#include <jpeglib.h>
+
+#endif /* IS_SDLv1 */
 
 #include "pgcompat.h"
 

